interp/constrextern.ml

```diff
@@ -1260,7 +1260,7 @@ and extern_notation (custom,scopes as allscopes) vars t rules =
         (* Try availability of interpretation ... *)
         match keyrule with
           | NotationRule (_,ntn as specific_ntn) ->
-             (match availability_of_entry_coercion custom (fst ntn) with
+             (match availability_of_entry_coercion custom InConstrEntrySomeLevel with
```

Comment:
Same here as above.

Reply:
At first view, this is precisely where you need to recover the `entry_level` part of the level found by looking in the `Notgram_ops.level_of_notation` table.